### ISSUE TITLE
Document <input type=checkbox switch>

### ISSUE THREAD

**Issue by annevk** [mdn/content#29231]
Once https://github.com/whatwg/html/pull/9546 is merged and implemented it'd be good to have documentation on MDN.

**Comment by Josh-Cena** [comment]
Hello, given the old age and relative immaturity and pushback on the feature, I don't think it is ready for documentation in the near future. We will be keeping an eye out for when it starts shipping in a browser (either Chrome or Firefox), but I think it's no longer tractable to track the spec status.

**Comment by annevk** [comment]
I'm not sure I understand. This has shipped in Safari and Ladybird and other than style-ability I'm not aware of any pushback.

It also seems like the policy on this might be inconsistent as https://developer.mozilla.org/en-US/docs/Web/API/Local_Font_Access_API for instance as about the same level of agreement (only shipped in Chrome and Firefox and Safari oppose it), but does have documentation.

**Comment by Josh-Cena** [comment]
Thanks. I didn't realize it has shipped to stable Safari. That makes it qualify our [criteria for inclusion](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/What_we_write#when_we_document_a_new_technology), I suppose; we need at least one stable implementation and a stable spec. Would you consider the spec (at least, the draft of it) to be stable enough?

**Comment by annevk** [comment]
Yes, modulo rebasing the PR against the latest HTML standard the text is stable.

**Comment by jensimmons** [comment]
And it shipped in [Safari 17.4](https://webkit.org/blog/15063/webkit-features-in-safari-17-4/) almost 18 months ago. 

We wrote an article about it here: https://webkit.org/blog/15054/an-html-switch-control/ 

Any policy or habit that considers features that are only-in-Chrome as web standards that other browser should ship, while features that are only-in-Safari as ideas that are a long way from being a web standard — this is not health for the web.

**Comment by Josh-Cena** [comment]
I had no intention to imply or claim that. Our standard is to document things that are in at least one stable browser—Safari is enough to qualify. My assumption that it's immature was purely based on the outspoken concerns from other browsers, and is neither based on research nor testing. Given that I have been corrected and it is indeed in production, sure we can document it.
